src/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixer.php

```diff
@@ -474,9 +477,54 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 }])
                 ->setDefault([])
                 ->getOption(),
+            (new FixerOptionBuilder('target', 'Target version of PHPUnit.'))
+                ->setAllowedTypes(['string'])
+                ->setAllowedValues([
+                    PhpUnitTargetVersion::VERSION_10_0,
+                    PhpUnitTargetVersion::VERSION_11_0,
+                    PhpUnitTargetVersion::VERSION_NEWEST,
+                ])
+                ->setDefault(PhpUnitTargetVersion::VERSION_10_0) // @TODO change to "VERSION_NEWEST" on next major 4.0
+                ->getOption(),
         ]);
     }
 
+    protected function configurePostNormalisation(): void
+    {
+        $dynamicMethods = [];
+        if (PhpUnitTargetVersion::fulfills($this->configuration['target'], PhpUnitTargetVersion::VERSION_11_0)) {
+            $dynamicMethods = [
+                'any',
+                'atLeast',
+                'atLeastOnce',
+                'atMost',
+                'exactly',
+                'never',
+                'once',
+                'throwException',
+            ];
+        }
+        if (PhpUnitTargetVersion::VERSION_11_0 === $this->configuration['target']) {
+            $dynamicMethods[] = 'onConsecutiveCalls';
```

Comment:
why this shall be added only on v11, but not on v10 nor v12 ?
is it really that method was added temporarily only for single PHPUnit version ?

Reply:
> is it really that method was added temporarily only for single PHPUnit version ?

Yes, this is verified in tests